common/enablers/epochFlags.go

```diff
@@ -90,6 +90,7 @@ type epochFlagsHolder struct {
 	maxBlockchainHookCountersFlag               *atomic.Flag
 	wipeSingleNFTLiquidityDecreaseFlag          *atomic.Flag
 	alwaysSaveTokenMetaDataFlag                 *atomic.Flag
+	changeUsernameFlag                          *atomic.Flag
```

Comment:
Maybe Name with caps N here, as it is used in the files above!

Reply:
done

Reply:
changed all to Username